smk-edit/static/components/tab-application.js

```diff
@@ -41,6 +41,23 @@ export default importComponents( [
                     this.$store.commit( 'configViewerBaseMap', val )
                 }
             },
+            esriApiKey: {
+                get: function () {
+                    return this.$store.getters.configViewerEsriApiKey
+                },
+                set: function ( val ) {
+                    this.location = null
+                    this.$store.commit( 'configViewerEsriApiKey', val )
+                }
+            },
+            selectedBasemapIsEsriType: function() {
+                for (const bm of baseMaps) {
+                    if(bm.id == this.$store.getters.configViewerBaseMap && bm.type == "ESRI") {
```

Comment:
Generally speaking with JavaScript, it is safer to use the 'strict equality operator' (===) instead of the 'abstract equality operator' (==). It probably won't make a difference here due to the nature of the operators and the fact that there are two parts to the if clause, but it is something to be aware of.